api/models/user.js

```diff
@@ -0,0 +1,14 @@
+const Sequelize = require('sequelize');
+
+module.exports = function(sequelize, DataTypes){
+    return sequelize.define('users', {
```

Comment:
You'll want to add columns to match this table: https://github.com/codelouisvilleproject/database-schema/blob/master/sql/V2__AddUsersAndActivities.sql#L3

We'll also need a PR on `database-schema` to add an `email` column.